cupy/__init__.py

```diff
@@ -262,6 +262,7 @@ def is_available():
 from cupy._manipulation.join import hstack  # NOQA
 from cupy._manipulation.join import stack  # NOQA
 from cupy._manipulation.join import vstack  # NOQA
+from cupy._manipulation.join import row_stack  # NOQA
```

Comment:
```suggestion
from cupy._manipulation.join import vstack as row_stack  # NOQA
```

This seems easier! (thanks @asi1024 )